src/components/pages/CourseSearch.js

```diff
@@ -0,0 +1,60 @@
+import React, { Fragment, useState, useEffect } from 'react';
+import Requestor from '../Requestor';
+
+const CourseSearch = () => {
+  const [courses, setCourses] = useState([]);
+
+  // Get courses
+  useEffect(() => {
+    getCourses();
+    // eslint-disable-next-line
+  });
+
+  const getCourses = async () => {
```

Comment:
This function should take in arguments `async ({ filters, offset, size } = DEFAULT_PARAMS)`